pkgs/top-level/all-packages.nix

```diff
@@ -21610,7 +21610,7 @@ in
     inherit (darwin.apple_sdk.frameworks) Security;
   };
 
-  zeronet = callPackage ../applications/networking/p2p/zeronet { };
+  zeronet = python3Packages.callPackage ../applications/networking/p2p/zeronet { };
```

Comment:
Shouldn't it be simply `callPackage` since this is a python application? (nixpkgs manual section 9.14.2.2.2)

Reply:
Correct. In case of an application the Python package set or interpreter should be passed in.

Reply:
We only want to pull from python3 packages in this case.

Reply:
That can be done by replacing `python2Packages` with `python3Packages` in `zeronet/default.nix`